api/modules/budgeting.js

```diff
@@ -142,6 +145,8 @@ const budgeting = module.exports = (() => {
 
             if (dateIncurredOverride) {
                 updatedAttributes.date_incurred = dateIncurredOverride
+            } else if (finalizedAt) {
+                updatedAttributes.date_incurrred = moment(finalizedAt).format()
```

Comment:
Fix typo: `date_incurrred` to `date_incurred`

Reply:
Let's also format it to be `format('YYYY-MM-DD')` since that is the same format used when filling out `date_incurred` from the Stripe metadata